### PR TITLE
Update ADVANCED.md

### PR DESCRIPTION
**Pull request by ElMoha943**
# Description
I really believe that we shouldnt recommend people to use outdated/less optimized server jars. There are really better jars like purpur, why using spigot/bukkit? Took some inspiration from [this](https://github.com/YouHaveTrouble/minecraft-optimization) and [this](https://github.com/KennyTV/list-of-shame)

## Checklist

- [x] My contributions follows the style guidelines of this project
- [x] I have performed a self-review of my own contributions (checked for spelling errors, proper linking, etc)
- [x] I have run linting on my branch before submitting the PR

### CODE REVIEW

**Comment by JustDoom** [comment]
we're not recommending them exactly we are listing all the ones that exist. if we need to make it clear not to use bukkit etc then we'll do that

**Comment by ElMoha943** [comment]
> we're not recommending them exactly we are listing all the ones that exist. if we need to make it clear not to use bukkit etc then we'll do that

yes that what i did :D

File: tutorials/en_us/JARS/ADVANCED.md [hunk 1]
```diff
@@ -2,20 +2,24 @@
...
 If you want to go a little more in-depth with JAR picking, you've come to the right place.
 
-## Most popular Spigot/Bukkit JARs are :
+## Most popular Spigot/Bukkit JARs (PLUGINS):
+
+### Recommended top picks:
 
 - [PaperMC](https://papermc.io/) - Most popular JAR for Spigot/Bukkit, quite fast and stable, they roll out small patches at least every 2 days or so, good choice if you're just starting with your MC server and don't have much past experience because it's fast and simple. Most if not all spigot plugins work with this JAR.
   starting with your MC server and don't have much past experience because it's fast and simple.
-- [Spigot](https://www.spigotmc.org/) - Original fork of Bukkit, improved performance vs CraftBukkit, still widely used, seems to be harder for newbies to download because it links to Jenkins,
```
Comment (ElMoha943) [inline tutorials/en_us/JARS/ADVANCED.md:9]: we should link to [getbukkit](https://getbukkit.org) then where people can download the jars more easy.

**Comment by SidhantUnnithan** [comment]
You are right but removing Spigot is not a good decision. This is a wiki, so all the major server jars are to be mentioned, regardless of whether they are optimized or not. Spigot is still not at a place where no one uses it. I am also a bit skeptical of giving out opinions in general. This wiki is just to give information, and not to give recommendations of any kind. Saying that it is not as optimized is one thing but outright recommending a server jar is not in the spirit of the wiki.

**Comment by ElMoha943** [comment]
> You are right but removing Spigot is not a good decision. This is a wiki, so all the major server jars are to be mentioned, regardless of whether they are optimized or not. Spigot is still not at a place where no one uses it. 

i understand that, but in a previous commit someone deleted yatopia saying it was bc its not a good jar, i think we can all agree that craftbukkit its not a good jar. We could leave all the jars there but we should tell people wich are the best/most optimized ones and advise them to use those over the others.

>I am also a bit skeptical of giving out opinions in general. This wiki is just to give information, and not to give recommendations of any kind. Saying that it is not as optimized is one thing but outright recommending a server jar is not in the spirit of the wiki.

 I believe is more than just an opinion that paper > spigot > bukkit. Its proven by facts and what i believe we are trying to do here is helping other server owners have better servers, optimization is one important fact that i consider should be adressed. We should **GIVE RECOMENDATIONS**, the users can follow them or not, thats up to them

Anyways, feel free to make the changes you feel necesary or rejecting the pull request :)

**Comment by SidhantUnnithan** [comment]
I didn't know about the yatopia PR and if that's true, I disagree with that PR too. 

Again, you're right. If one server jar is more optimised than another one, proven with facts, then it should definitely be mentioned. However I'll reiterate - there's a difference between "paper > bukkit based on numbers/popularity" and "we personally recommend paper because it is better."

The former is fine, the latter shouldn't be. The former is backed by facts. The latter is backed by opinions. Its a fine line that is easy to cross, and that's why PR reviews exist :)

Anything backed by opinions is not consistent with the spirit of any wiki really, and if this wiki is to be completely unbiased and a one stop for all information portal/guide, I think the latter in the above example should be avoided completely.

However, of course the direction in which this wiki is headed is up for debate. @JustDoom what do you think?

**Comment by ElMoha943** [comment]
thanx you for your words, i invite you to suggest any changes you feel necessary :)

**Comment by ElMoha943** [comment]
@SidhantUnnithan i made some changes, let me know what you think!

**Comment by ElMoha943** [comment]
oof i accidentally closed it xD

**Comment by JustDoom** [comment]
> I didn't know about the yatopia PR and if that's true, I disagree with that PR too.
> 
> Again, you're right. If one server jar is more optimised than another one, proven with facts, then it should definitely be mentioned. However I'll reiterate - there's a difference between "paper > bukkit based on numbers/popularity" and "we personally recommend paper because it is better."
> 
> The former is fine, the latter shouldn't be. The former is backed by facts. The latter is backed by opinions. Its a fine line that is easy to cross, and that's why PR reviews exist :)
> 
> Anything backed by opinions is not consistent with the spirit of any wiki really, and if this wiki is to be completely unbiased and a one stop for all information portal/guide, I think the latter in the above example should be avoided completely.
> 
> However, of course the direction in which this wiki is headed is up for debate. @JustDoom what do you think?

Im trying to keep opinions about plugins, server jars etc out of this. a lot of preferences about server jars is your opinion though

**Comment by ElMoha943** [comment]
> > I didn't know about the yatopia PR and if that's true, I disagree with that PR too.
> > Again, you're right. If one server jar is more optimised than another one, proven with facts, then it should definitely be mentioned. However I'll reiterate - there's a difference between "paper > bukkit based on numbers/popularity" and "we personally recommend paper because it is better."
> > The former is fine, the latter shouldn't be. The former is backed by facts. The latter is backed by opinions. Its a fine line that is easy to cross, and that's why PR reviews exist :)
> > Anything backed by opinions is not consistent with the spirit of any wiki really, and if this wiki is to be completely unbiased and a one stop for all information portal/guide, I think the latter in the above example should be avoided completely.
> > However, of course the direction in which this wiki is headed is up for debate. @JustDoom what do you think?
> 
> Im trying to keep opinions about plugins, server jars etc out of this. a lot of preferences about server jars is your opinion though

if you dont like opinions you should check FINDING.md i think it has a lot of them, also what are you going to do with this pr? and could you also check my pr about the optimization md? thanx in advance

**Comment by SidhantUnnithan** [comment]
In my opinion this PR looks fine now. The last commit is free of opinions. And @ElMoha943, I'll check out FINDING.md tonight and weed out opinions. @JustDoom, I suggest you take a final look at the last 2 commits and if it looks fine to you too, merge it.

**Review by JustDoom — APPROVED** [review]

**Pull request merged**